src/Illuminate/Database/Schema/Columns/Column.php

```diff
@@ -0,0 +1,286 @@
+<?php
+
+namespace Illuminate\Database\Schema\Columns;
+
+/**
+ * Class Column
+ *
+ * @property-read string $type
+ * @property-read string $name
+ * @property-read string $after
+ * @property-read string $comment
+ * @property-read mixed $default
+ * @property-read bool $first
+ * @property-read bool $nullable
+ * @property-read string $storedAs
+ * @property-read string $virtualAs
+ * @property-read string|true $primary
+ * @property-read string|true $unique
+ * @property-read string|true $index
+ * @property-read string|true $spatialIndex
+ */
+class Column
+{
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $after;
+
+    /**
+     * @var string
+     */
+    private $comment;
+
+    /**
+     * @var mixed
+     */
+    private $default;
+
+    /**
+     * @var bool
+     */
+    private $first = false;
+
+    /**
+     * @var bool
+     */
+    private $nullable = false;
+
+    /**
+     * @var string
+     */
+    private $storedAs;
+
+    /**
+     * @var string
+     */
+    private $virtualAs;
+
+    /**
+     * @var bool
+     */
+    private $change = false;
+
+    /**
+     * @var string|true
+     */
+    private $primary;
+
+    /**
+     * @var string|true
+     */
+    private $unique;
+
+    /**
+     * @var string|true
+     */
+    private $index;
+
+    /**
+     * @var string|true
+     */
+    private $spatialIndex;
+
+    /**
+     * Column constructor.
+     *
+     * @param string $type
+     * @param string $name
+     */
+    public function __construct(string $type, string $name)
+    {
+        $this->type = $type;
+        $this->name = $name;
+    }
+
+    /**
+     * @param string $name
+     * @return mixed
+     */
+    public function __get(string $name)
+    {
+        return $this->{$name};
+    }
+
+    /**
+     * @param string $name
+     * @param mixed $value
+     */
+    public function __set(string $name, $value)
+    {
+        // make all properties read only
+        throw new \RuntimeException("Trying to write to read only property '$name'");
+    }
+
+    /**
+     * @param string $name
+     * @return bool
+     */
+    public function __isset(string $name): bool
+    {
+        return isset($this->{$name});
+    }
+
+    /**
+     * @return array
+     */
+    public function getAttributes(): array
+    {
+        return get_object_vars($this);
+    }
+
+    /**
+     * Place the column "after" another column (MySQL)
+     *
+     * @param string $column
+     * @return Column
+     */
+    public function after(string $column): Column
+    {
+        $this->after = $column;
+        return $this;
+    }
+
+    /**
+     * Add a comment to a column (MySQL)
```

Comment:
These database specific comments are problematic, for example: postgres supports comments on columns too.

Reply:
I copied these comments from docs https://github.com/laravel/docs/blob/master/migrations.md#column-modifiers
Postgres supports comments but laravel schema build does not support them for postgres and for other DBs.